### PR TITLE
Product fixtures and merchant-order relationship update

### PR DESCRIPTION
**Pull request by raisahv**
- Removed one Merchant to many Orders relationship in Merchant and Order models
- Added product fixtures
- All tests passing

### CODE REVIEW

File: app/models/merchant.rb [hunk 1]
```diff
@@ -1,6 +1,5 @@
 class Merchant < ApplicationRecord
-  has_many :orders
-  has_many :products
+  has_many :products, dependent: :destroy
```
Comment (dtingg) [inline app/models/merchant.rb:3]: Do we really want to delete those or should be use nullify? 
has_many :products, dependent: :nullify

File: app/models/product.rb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 class Product < ApplicationRecord
   belongs_to :merchant
-  has_many :orderitems
-  has_many :reviews
+  has_many :orderitems, dependent: :destroy
```
Comment (dtingg) [inline app/models/product.rb:3]: Should this one be nullify too?

**Review by dtingg — COMMENTED** [review]
Should we destroy dependents or just set them to null, so we can still see the order?

File: app/models/merchant.rb (hunk 1 above)
Comment (raisahv) [reply app/models/merchant.rb:3]: Sure we can use nullify, I'm up for either

**Review by KKennedyCodes — APPROVED** [review]

**Review by NatalieTapias — COMMENTED** [review]
lgtm

**Pull request merged**